packages/dd-trace/src/profiling/config.js

```diff
@@ -87,6 +90,30 @@ function ensureExporters (exporters, options) {
   return exporters
 }
 
+function getProfiler (name, options) {
+  switch (name) {
+    case 'cpu':
+      return new CpuProfiler(options)
+    case 'heap':
+      return new HeapProfiler(options)
+  }
+}
+
+function ensureProfilers (profilers, options) {
+  if (typeof profilers === 'string') {
+    profilers = profilers.split(',')
+  }
+
+  for (let i = 0; i < profilers.length; i++) {
+    const profiler = profilers[i]
+    if (typeof profiler === 'string') {
+      profilers[i] = getProfiler(profiler, options)
```

Comment:
This will assign `undefined` if the name is not `cpu` or `heap`.

Reply:
It might make sense to also sanitize the values further to make sure that incorrect usage doesn't result in an error.